frontend/src/data/sorting/AgeSorterStrategy.ts

```diff
@@ -0,0 +1,58 @@
+import { Breakdowns } from "../query/Breakdowns";
+import { Row } from "../utils/DatasetTypes";
+import { AbstractSortStrategy } from "./AbstractDataSorter";
+
+export class AgeSorterStrategy extends AbstractSortStrategy {
+  frontValues: string[];
+  backValues: string[];
+
+  appliesToBreakdowns = (b: Breakdowns) => {
+    return b.hasOnlyAge();
+  };
+
+  constructor(frontValues: string[] = [], backValues: string[] = []) {
+    super();
+    // We Reverse these becauase we want the human readable version of sorting.
+    // eg. [A, B, C] to work with the index calculation. so that A is first and B is
+    // second ... ect
+    this.frontValues = frontValues.reverse();
+    this.backValues = backValues;
+  }
+
+  readonly compareFn = (l: Row | string, r: Row | string) => {
+    let lAge = typeof l === "string" ? l : l["age"];
```

Comment:
Reminder to replace `let` with `const` when the variable is never reassigned, here and throughout the PR